src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/PlanTypeOptions/PlanTypeOptions.spec.tsx

```diff
@@ -265,40 +265,6 @@ describe('PlanTypeOptions', () => {
         expect(teamBtn).not.toHaveClass('bg-ds-primary-base')
       })
 
-      describe('plan param is set to team', () => {
```

Comment:
This logic was duplicated unnecessarily so I removed it.

Reply:
love it